core/src/navigation/LeftNav.svelte

```diff
@@ -273,12 +280,65 @@
     semiCollapsibleButton = LuigiConfig.getConfigValue('settings.responsiveNavigation') === 'semiCollapsible';
   });
 
+  function convertEntriesToToolLayout(entries) {
+    if (btpToolLayout) {
```

Comment:
early return?

Reply:
Since there are different opinions out there if early return is good or bad, I would leave it like it is ;)